apps/bfd-pipeline/bfd-pipeline-shared-test-utils/src/main/java/gov/cms/bfd/pipeline/CleanupTestUtils.java

```diff
@@ -0,0 +1,291 @@
+package gov.cms.bfd.pipeline;
+
+import gov.cms.bfd.DatabaseTestUtils;
+import gov.cms.bfd.model.rda.Mbi;
+import gov.cms.bfd.model.rda.entities.RdaFissClaim;
+import gov.cms.bfd.model.rda.entities.RdaFissDiagnosisCode;
+import gov.cms.bfd.model.rda.entities.RdaFissPayer;
+import gov.cms.bfd.model.rda.entities.RdaFissProcCode;
+import gov.cms.bfd.model.rda.entities.RdaFissRevenueLine;
+import gov.cms.bfd.model.rda.entities.RdaMcsClaim;
+import gov.cms.bfd.model.rda.entities.RdaMcsDetail;
+import gov.cms.bfd.model.rda.entities.RdaMcsDiagnosisCode;
+import java.math.BigDecimal;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.Persistence;
+import javax.sql.DataSource;
+import lombok.Getter;
+
+/**
+ * Utility class for supporting testing of the CleanupJob tasks. (Borrows from
+ * gov.cms.bfd.server.war.utils.RDATestUtils.)
+ */
+public class CleanupTestUtils {
+
+  /** Tracking entities (tables) so they can be cleaned after. */
+  private static final List<Class<?>> TABLE_ENTITIES =
+      List.of(
+          RdaFissRevenueLine.class,
+          RdaFissPayer.class,
+          RdaFissDiagnosisCode.class,
+          RdaFissProcCode.class,
+          RdaFissClaim.class,
+          RdaMcsDetail.class,
+          RdaMcsDiagnosisCode.class,
+          RdaMcsClaim.class,
+          Mbi.class);
+
+  /** Path to use for persistence units. */
+  public static final String PERSISTENCE_UNIT_NAME = "gov.cms.bfd.rda";
+
+  /** Test mbi. */
+  public static final String MBI = "123456MBI";
+
+  /** Test mbi hash. */
+  public static final String MBI_HASH = "a7f8e93f09";
+
+  /** Test fiss claim (DCN). */
+  public static final String FISS_CLAIM_DCN = "123456d";
+
+  /** EntityManager to use with tests. */
+  private EntityManager entityManager;
+
+  /** EntityManagerFactory to use with tests. */
+  @Getter private EntityManagerFactory entityManagerFactory;
+
+  /** Initializes the test utility. */
+  public void init() {
+    final DataSource dataSource = DatabaseTestUtils.get().getUnpooledDataSource();
+
+    final Map<String, Object> hibernateProperties =
+        Map.of(org.hibernate.cfg.AvailableSettings.DATASOURCE, dataSource);
+
+    entityManagerFactory =
+        Persistence.createEntityManagerFactory(PERSISTENCE_UNIT_NAME, hibernateProperties);
+
+    entityManager = entityManagerFactory.createEntityManager();
+  }
+
+  /** Closes all resources. */
+  public void close() {
+    if (entityManager != null) {
+      entityManager.close();
+    }
+  }
```

Comment:
With TransactionManager this would not be necessary.

Reply:
Removed